sample-kotlin-model/src/main/kotlin/com/vimeo/dummy/sample_kotlin/KotlinSamples.kt

```diff
@@ -0,0 +1,25 @@
+package com.vimeo.dummy.sample_kotlin
+
+import com.google.gson.annotations.SerializedName
+import com.vimeo.stag.UseStag
+
+/**
+ * Test kotlin model
+ *
+ * Created by restainoa on 5/8/17.
+ */
+@UseStag
+class KotlinSamples {
+    var stringField: String? = null
+
+    var nonNullStringField: String = "default"
+
+    var intField: Int? = null
+
+    var longField: Long = 1
+
+    @SerializedName("boolean_field")
+    var booleanField: Boolean? = null
+
+    var testField: Any? = null
```

Comment:
I'm going to ticket this after merge, but what I would like to do is have a separate `stag-library-compiler-kotlin` module that generates kotlin code instead of java. This would allow us to support the module level visibility without needing to jump through hoops to access the fields. It isn't straight forward though as it will require major refactoring because code gen is baked into the `stag-library-compiler` module and we need code gen to be interchangeable in order to support kotlin generation.